tooling/redhat-camel-spring-boot-bom/pom.xml

```diff
@@ -144,7 +144,7 @@
             <dependency>
                 <groupId>org.infinispan</groupId>
                 <artifactId>infinispan-bom</artifactId>
-                <version>14.0.11.Final</version>
+                <version>14.0.13.Final</version>
```

Comment:
@Croway is it fine to use this one? IIRC you changed this version recently, thanks

Reply:
good catch! @cunningt we have to align with quarkus, and use 14.0.11.Final

Reply:
Aligned to infinispan 14.0.11.Final.    I think what is happening here is that our prod branch is 4.0.0-SNAPSHOT but there's also an upstream 4.0.0-SNAPSHOT camel-spring-boot that points to 4.0.0-SNAPSHOT upstream apache, which uses 14.0.13.Final. 

In PNC, everything should be fine and I think infinispan-version will be picked up from the camel tage we build because we're not dealing at all with SNAPSHOTs, but if you run the build on the branch locally I think it will try to resolve infinispan-version to 14.0.13.Final because that is the version in 4.0.0.